script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -116,8 +122,8 @@ The priest nods, embarrassed.
 -- Page 8, Offset 41403.
 -- 「どうして転生なんかを？　一個体として生き続けた方が簡単だし、なにより人に迷惑をかけないでしょうに」
 "Why reincarnation, anyway?
- It'd be a lot easier to focus on prolonging your life as an individual,
- and more importantly, you wouldn't go around causing problems for other people."
+ It'd be a lot easier to to prolong your own individual life,
```

Comment:
Deja vu striking me rn

I don't really feel like the difference exists, once again. Being himself distinct to other people is what the point is. Reincarnation involves other people, that he causes problems for... etc

Reply:
It's an awkward distinction to elaborate on, since this is a fairly uncommon use case in The Real World, but I do feel it exists - individual life is individual defined as different, separate, his own existence, but that crucially _does not_ negate him reincarnating into other people - he still exists as an 'individual life' when he does that, there is still the tangible portion that is Roa. Life _as an individual_ is him as as a flesh and blood organism - a single individual defined as a person, and one that doesn't impinge on others

Reply:
I'll be honest, that just convinced me "own individual life" is preferrable. "as an individual" sounds too impersonal. Maybe if it was "as a single individual", though that sounds a bit unromantic. Given the JP, I suppose that wouldn't be the biggest deal, but still.

Reply:
Yeah I agree it's awkward to be explicit without being clunky. Annoyingly 'person', 'individual', etc. are overloaded. Oh well. People will work it out.